src/Controller/DefaultController.php

```diff
@@ -1248,7 +1236,6 @@ public function sortJobsAction(Request $request)
                 array(),
                 array('link' => $this->generateUrl('showClients'))
             );
-            $this->getDoctrine()->getManager()->flush();
             $this->get('session')->getFlashBag()->add(
```

Comment:
We should flush something if we have reordered jobs? (I assume DB was updated, although I see no persist?

Reply:
I think there is no change in the DB When sorting jobs, as it only makes a query with an "orderBy" statement to sort the jobs by priority and show them in the form.

Reply:
POST action is for saving the priority (order) of jobs. Must be saving to DB but can't see where really.

Reply:
I have tested it, and the jobs are been reordered and the priority is saved in the DB correctly.